pkg/cmd/cli/cmd/edit.go

```diff
@@ -23,14 +22,34 @@ import (
 	"github.com/openshift/origin/pkg/util/jsonmerge"
 )
 
+//EditOptions is a struct that contains all variables needed for cli edit command.
+//These variables are used in Complete, Validate and RunEdit methods.
+type EditOptions struct {
+	out       io.Writer
+	printer   kubectl.ResourcePrinter
+	namespace string
+	infos     []*resource.Info
+	results   *editResults
+	rmap      *resource.Mapper
+	f         *clientcmd.Factory
+	cmd       *cobra.Command
```

Comment:
Neither this. Instead populate o.out with cmd.Out() and use that instead in RunEdit